Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/PanicMod.java

```diff
@@ -25,17 +25,27 @@ public void onEnable()
 		wurst.events.add(UpdateListener.class, this);
 	}
 	
+	private Mod panicedMods[] = [];
+	
 	@Override
 	public void onUpdate()
 	{
 		for(Mod mod : wurst.mods.getAllMods())
 			if(mod.getCategory() != Category.HIDDEN && mod.isEnabled())
+			{
 				mod.setEnabled(false);
+				panicedMods.add(mod);
+			}
 	}
 	
 	@Override
 	public void onDisable()
 	{
+		for(Mod mod : wurst.mods.getAllMods())
+		{//I like curly braces :}
+			mod.setEnabled(true);
+			panicedMods.add(mod);

```

Comment:
This is not going to work either. Please change it to loop through `panicedMods` instead of all mods and then do `panicedMods.clear()`.